pkg/client/monitoring/v1/zz_generated.deepcopy.go

```diff
@@ -986,3 +995,132 @@ func (in *TLSConfig) DeepCopy() *TLSConfig {
 	in.DeepCopyInto(out)
 	return out
 }
+
+// DeepCopyInto is an autogenerated deepcopy function, copying the receiver, writing into out. in must be non-nil.
+func (in *ThanosGCSSpec) DeepCopyInto(out *ThanosGCSSpec) {
+	*out = *in
+	if in.Bucket != nil {
+		in, out := &in.Bucket, &out.Bucket
+		if *in == nil {
+			*out = nil
+		} else {
+			*out = new(string)
+			**out = **in
+		}
+	}
+	return
+}
+
+// DeepCopy is an autogenerated deepcopy function, copying the receiver, creating a new ThanosGCSSpec.
+func (in *ThanosGCSSpec) DeepCopy() *ThanosGCSSpec {
+	if in == nil {
+		return nil
+	}
+	out := new(ThanosGCSSpec)
+	in.DeepCopyInto(out)
+	return out
+}
+
+// DeepCopyInto is an autogenerated deepcopy function, copying the receiver, writing into out. in must be non-nil.
```

Comment:
`writing into out. in must be non-nil.`

`In must be non-nil` instead? `In` first letter uppercase? I know this is autogenerated, but..

Reply:
yeah we should report these to the kubernetes/code-generator repo